src/features/warpCore/warpCoreConfig.ts

```diff
@@ -57,7 +57,7 @@ function filterToIds(
 function dedupeTokens(tokens: WarpCoreConfig['tokens']): WarpCoreConfig['tokens'] {
   const idToToken: Record<string, WarpCoreConfig['tokens'][number]> = {};
   for (const token of tokens) {
-    const id = `${token.chainName}|${token.addressOrDenom?.toLowerCase()}`;
+    const id = `${token.chainName}|${token.addressOrDenom?.toLowerCase()}|${token.standard.toLowerCase()}`;
```

Comment:
we need to also differentiate by standard, to permit a token to be birdged via WR and/or intents